gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceManager.java

```diff
@@ -218,6 +211,12 @@ public GobblinServiceManager(String serviceName, String serviceId, Config config
       this.helixManager = Optional.absent();
     }
 
+    // Initialize DagManager
```

Comment:
Initially `DagManager` is only initialized when `isFlowCatalogEnabled` set to true, but moving it out means this condition doesn't apply anymore.  Is that on purpose ?

Reply:
Thanks for noticing this! The DagManager initialization does not depend on whether FlowCatalog is enabled or not. We have a separate config to enable/disable DagManager. I can't recall why it was inside if (isFlowCatalogEnabled). Yes, the move outside the if block is intentional.